jetbrains-rider/build.gradle.kts

```diff
@@ -175,11 +170,21 @@ val prepareNuGetConfig = tasks.register("prepareNuGetConfig") {
     doLast {
         val nugetPath = getNugetPackagesPath()
         val configText = """<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <packageSources>
-    <add key="resharper-sdk" value="$nugetPath" />
-  </packageSources>
-</configuration>
+  <configuration>
+    <packageSources> 
+    ${
+        if (codeArtifactNugetUrl.isPresent) {
+            """
+       |   <clear /> 
+   |       <add key="codeartifact-nuget" value="${codeArtifactNugetUrl.get()}v3/index.json" />
```

Comment:
```suggestion
       |   <clear /> 
       |   <add key="codeartifact-nuget" value="${codeArtifactNugetUrl.get()}v3/index.json" />
```

Reply:
For the suggested change L 178 & L 179. 
Its not generating a proper xml format in Nuget.config file. 
Its generating in this way: 
<config>
   <pkg>
        <clear/>
    <add key= ""     />    
         <add ..................  />
   </pkg>
</config>